ideas/121-swarm-compensation.md

```diff
@@ -0,0 +1,79 @@
+---
+id: 121-swarm-compensation
+title: Swarm Compensation Experiments
+status: in-progress
+created: 2018-04-02
+---
+
+## Preamble
+
+    Idea: 121-swarm-compensation
+    Title: Swarm Compensation
+    Status: In Progress
+    Created: 2018-04-02
+    Related: 120-swarm-process
+
+## Summary
+Ultimately we would like Status to become a public good, owned, developed and ran by it's very own users. In order to achieve that we need to open up our processes for contributions from "the outside" and establish ways for users to propose & incentivise projects they see as important.
+
+Swarms are our current attempt at enabling this. They are ephemeral teams forming to achieve a specified goal and disbanding once achieved or failure becomes apparent. Their defined lifetime and -cycle encourage constant re-evaluation &. evolution.
+
+Compensating contributors for their work is an important component of this and where `120-swarm-process` focuses on establishing a baseline of processes that are open to everyone this idea will focus on experimentally devising compensation mechanisms. 
+
+## Swarm Participants
+
+- Lead Contributor: @martinklepsch
+- Contributor ('payment'): @jason / @carl
+- Contributor: @oskarth
+- Evaluation ('compensation criteria'): Voting mechanism (Polly survey)
```

Comment:
Evaluation by the members who participated in the experimentation I assume?

Reply:
Right this should be noted will update.